app/gui2/src/components/GraphEditor/GraphNode.vue

```diff
@@ -430,7 +430,7 @@ const documentation = computed<string | undefined>({
     <div class="binding" @pointerdown.stop v-text="node.pattern?.code()" />
     <button
       v-if="!menuVisible && isRecordingOverridden"
-      class="overrideRecordButton"
+      class="overrideRecordButton recording-overridden"
```

Comment:
lets use `data-testid` instead of classes when all you want to do is to locate a thing in tests. It is even nicely supported by playwright: https://playwright.dev/docs/locators#locate-by-test-id